src/client/core/utils/dom.js

```diff
@@ -443,3 +443,9 @@ export function findIframeByWindow (iframeWindow, iframeDestinationWindow) {
 
     return null;
 }
+
+export function canAcceptValue (element) {
```

Comment:
The word `editor`  has a narrower meaning (more specified) in my opinion. 
@VasilyStrelyaev, can we use the word "intake" ?